src/Tests/Microsoft.NET.Restore.Tests/GivenThatWeWantToRestoreProjectsUsingNuGetConfigProperties.cs

```diff
@@ -156,5 +159,52 @@ private void ExtractNupkg(string nugetCache, string nupkg)
                 }
             }
         }
+
+        private class MockPackageDownloader : IPackageDownloader
```

Comment:
This seems unrelated.

Reply:
This is unrelated to the feature.  I retargeted this to master, and tests targeting .NET Core 2.1 were failing.  So I'm trying to update the stage 0 CLI, which brought in an update to NuGet, which had API changes which required this addition... 🤷‍♂️